src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -496,9 +496,11 @@ def __init__(
         "o1-mini-2024-09-12",
         "o1-preview",
         "o1-preview-2024-09-12",
+        "o3-mini",
+        "o3-mini-2025-01-31",
     ],
 )
-class OpenAIO1StreamingClient(OpenAIStreamingClient):
+class OpenAIReasoningStreamingClient(OpenAIStreamingClient):
```

Comment:
how can this be re-named and not require any code canges?

Reply:
the class itself is registered via the decorator which maps playground model names to the client they use